test/AxelarGatewaySinglesig.js

```diff
@@ -1311,5 +1310,59 @@ describe('AxelarGatewaySingleSig', () => {
           });
       });
     });
+
+    describe('send token from gateway', () => {
```

Comment:
same here

Reply:
We should test both types of internal tokens.